kernel/classes/datatypes/ezdatetime/ezdatetimetype.php

```diff
@@ -592,7 +600,9 @@ function unserializeContentObjectAttribute( $package, $objectAttribute, $attribu
         $dateTimeNode = $attributeNode->getElementsByTagName( 'date_time' )->item( 0 );
         if ( is_object( $dateTimeNode ) )
         {
-            $timestamp = eZDateUtils::textToDate( $dateTimeNode->textContent );
+            $timestamp = eZTimestamp::getUtcTimestampFromLocalTimestamp(
+                eZDateUtils::textToDate( $dateTimeNode->textContent )
```

Comment:
This method is terribly named. But I can't blame you for that ;) I had to verify that it actually returns a timestamp.

Reply:
Yes it is :rofl: